src/locale/localed-util.c

```diff
@@ -243,10 +243,10 @@ int vconsole_read_data(Context *c, sd_bus_message *m) {
         return parse_env_file_fd(fd, "/etc/vconsole.conf",
                                  "KEYMAP",        &c->vc_keymap,
                                  "KEYMAP_TOGGLE", &c->vc_keymap_toggle,
-                                 "XKB_LAYOUT",    &c->x11_from_vc.layout,
-                                 "XKB_MODEL",     &c->x11_from_vc.model,
-                                 "XKB_VARIANT",   &c->x11_from_vc.variant,
-                                 "XKB_OPTIONS",   &c->x11_from_vc.options);
+                                 "XKBLAYOUT",     &c->x11_from_vc.layout,
+                                 "XKBMODEL",      &c->x11_from_vc.model,
+                                 "XKBVARIANT",    &c->x11_from_vc.variant,
+                                 "XKBOPTIONS",    &c->x11_from_vc.options);
```

Comment:
The new names are much less readable. But I agree that compat with Debian is important.